### DISCUSSION TITLE
[@types/node-fetch-cache] has wrong types for MemoryCache.set

### DISCUSSION THREAD

**Discussion by broksonic21** [DefinitelyTyped/DefinitelyTyped#67044] [Issues with a @types package]
The code has https://github.com/mistval/node-fetch-cache/blob/master/src/classes/caching/memory_cache.js#L37
async set(key, bodyStream, metaData)

but @types/node-fetch-cache has

set(value: any): Promise<any>;

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node-fetch-cache/index.d.ts#L26

[a comment by an automated account is not shown]

**Comment by broksonic21** [comment]
Also mentioned here: https://github.com/mistval/node-fetch-cache/issues/36

**Comment by fujikky** [comment]
@broksonic21  You are right. I had the wrong argument for the `set` method when I created the type.
I created a PR to fix it! https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67070

**Comment by broksonic21** [comment]
That PR fixes the issue I filed - thanks! I gave it an approval.
